<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomConfigDialog</class>
 <widget class="QDialog" name="RoomConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Room Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tileset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_TilesetID"/>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_BGLayerEnable">
         <property name="text">
          <string>Enable Background Layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Background Layer:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_BGLayerPicker"/>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_BGLayerScrolling">
         <property name="text">
          <string>Background Scrolling Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0Enable">
         <property name="text">
          <string>Enable Layer 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0Alpha">
         <property name="text">
          <string>Layer 0 Alpha Enable:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Alhpa Blend Attributes:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_AlphaBlendAttribute"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Layer 0 Mapping Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_Layer0MappingType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Use existing Layer 0: (only when Layer Mapping type is 0x20)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_Layer0Picker"/>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0Scrolling">
         <property name="text">
          <string>Layer 0 Scrolling Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer2Enable">
         <property name="text">
          <string>Enable Layer 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Layer Priority:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ComboBox_LayerPriority"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Room Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_RoomWidth">
         <property name="minimum">
          <number>18</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Room Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="SpinBox_RoomHeight">
         <property name="minimum">
          <number>14</number>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Details: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="graphicsView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RoomConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RoomConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
